config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml

```diff
@@ -21,60 +17,27 @@ spec:
     - ovn
     - libvirt
     - nova
-  baremetalSetTemplate:
-    deploymentSSHSecret: dataplane-ansible-ssh-private-key-secret
-    bmhLabelSelector:
-      app: openstack
-    ctlplaneInterface: enp1s0
-    dnsSearchDomains:
-      - osptest.openstack.org
+  deployStrategy:
+      deploy: true
   nodeTemplate:
+    ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
     nodes:
        edpm-compute-0:
          hostName: edpm-compute-0
-         ansible:
-           ansibleHost: 192.168.111.100
-           ansibleVars:
-             ctlplane_ip: 192.168.111.100
-             internal_api_ip: 172.17.0.100
-             storage_ip: 172.18.0.100
-             tenant_ip: 172.19.0.100
-             fqdn_internal_api: edpm-compute-0.example.com
-         networks:
-         - name: CtlPlane
-           subnetName: subnet1
-           defaultRoute: true
-           fixedIP: 192.168.111.100
-         - name: InternalApi
-           subnetName: subnet1
-         - name: Storage
-           subnetName: subnet1
-         - name: Tenant
-           subnetName: subnet1
-       edpm-compute-1:
-         hostName: edpm-compute-1
-         ansible:
-           ansibleHost: 192.168.111.101
-           ansibleVars:
-             ctlplane_ip: 192.168.111.101
-             internal_api_ip: 172.17.0.101
-             storage_ip: 172.18.0.101
-             tenant_ip: 172.19.0.101
-             fqdn_internal_api: edpm-compute-1.example.com
          networks:
-         - name: CtlPlane
-           subnetName: subnet1
-           defaultRoute: true
-           fixedIP: 192.168.111.101
-         - name: InternalApi
-           subnetName: subnet1
-         - name: Storage
-           subnetName: subnet1
-         - name: Tenant
-           subnetName: subnet1
+          - name: CtlPlane
+            subnetName: subnet1
+            defaultRoute: true
+            fixedIP: 192.168.122.100
+          - name: InternalApi
+            subnetName: subnet1
+          - name: Storage
+            subnetName: subnet1
+          - name: Tenant
+            subnetName: subnet1
     managementNetwork: ctlplane
     ansible:
-      ansibleUser: cloud-admin
+      ansibleUser: root
```

Comment:
Aren't we sticking with cloud-admin?

Reply:
I know we use cloud-admin for baremetal, but I believe install_yamls still using root

Reply:
We should probably decide what we actually want here. With tripleo, we went through the stages of `heat-admin` and `tripleo-admin`. I guess we want to standardise on something similar here? But probably not root right?

Understand the install_yamls situation. Just saying that we probably want to aim for a non-root user before we GA.